Source/WebCore/Modules/modern-media-controls/controls/macos-fullscreen-media-controls.css

```diff
@@ -74,6 +75,9 @@
     top: 18px;
 }
 
+.media-controls.mac.fullscreen .buttons-container.left .slider {
+    width: 60px;
+}
```

Comment:
Does this change the appearance of the fullscreen controls on platforms that don't support Liquid Glass?